crates/grpc-util/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "restate-grpc-util"
```

Comment:
Could that be part of restate-network or would that be needed in crates that don't depend on restate-network already?

Reply:
Currently, the `metadata-store` crate needs this functionality and does not depend on `restate-network`.